### PR TITLE
Adding missing test project config to .sln file

### LINKED ISSUE #33671: System.Runtime.WindowsRuntime.sln refuses to open
It just hangs here forever:
image

cc: @Anipik, @ViktorHofer, @safern

### PR DESCRIPTION
**Pull request by Anipik**
Fixes https://github.com/dotnet/runtime/issues/33671

I verified the repro and the fix locally.

### CODE REVIEW

File: src/libraries/System.Runtime.WindowsRuntime/System.Runtime.WindowsRuntime.sln [hunk 1]
```diff
@@ -26,6 +26,10 @@ Global
 		Release|Any CPU = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{C4854B44-ABFE-4BB5-8F89-F35FE6201338}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{C4854B44-ABFE-4BB5-8F89-F35FE6201338}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{C4854B44-ABFE-4BB5-8F89-F35FE6201338}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{C4854B44-ABFE-4BB5-8F89-F35FE6201338}.Release|Any CPU.Build.0 = Release|Any CPU
```
Comment (stephentoub) [inline src/libraries/System.Runtime.WindowsRuntime/System.Runtime.WindowsRuntime.sln]: Do we know why VS hangs without this?  Are the right folks aware of that bug?

**Review by stephentoub — APPROVED** [review]

File: src/libraries/System.Runtime.WindowsRuntime/System.Runtime.WindowsRuntime.sln (hunk 1 above)
Comment (Anipik) [reply src/libraries/System.Runtime.WindowsRuntime/System.Runtime.WindowsRuntime.sln]: Yes there is a similar issue opened in the VS https://github.com/dotnet/project-system/issues/5878

The fix for vs project was same here https://github.com/dotnet/runtime/pull/31950

**Review by ViktorHofer — APPROVED** [review]

**Comment by ViktorHofer** [comment]
@Anipik can you please check if there are other solutions which are missing configurations?

**Comment by Anipik** [comment]
> @Anipik can you please check if there are other solutions which are missing configurations?

yes i am already on working on it :P

**Review by safern — COMMENTED** [review]
LGTM

**Comment by stephentoub** [comment]
System.Runtime.WindowsRuntime.UI.Xaml appears to have the same problem 😦 
image

**Comment by stephentoub** [comment]
I fixed System.Runtime.WindowsRuntime.UI.Xaml.

**Comment by Anipik** [comment]
Just an update, i wrote a small program to check our solution files for this bug. And our solutions should no longer have this problem

```c#
        static void Main(string[] args)
        {
            Console.WriteLine("Hello World!");
            string[] slnFiles = Directory.GetFiles(@"C:\git\runtime\src\libraries", "*.sln", SearchOption.AllDirectories);
            foreach (var item in slnFiles)
            {
                string[] csProjs = Directory.GetFiles(Path.GetDirectoryName(item), "*.csproj", SearchOption.AllDirectories);
                if (!checkSln(item, csProjs.Length))
                    Console.WriteLine(item + " " + csProjs.Length.ToString());
            }
        }

        static bool checkSln(string sln, int num)
        {
            string[] lines = File.ReadAllLines(sln);
            int count = 0;
            foreach (var item in lines)
            {
                if (item.Contains(".Debug|Any CPU.ActiveCfg = Debug|Any CPU"))
                    count++;
            }
            return count == num;
        }
```

This code could report some false positives but they were really few in num (around 10) so i just checked those sln manually

**Pull request merged**